gp2gp-translator/src/integrationTest/resources/e2e-mapping/output-json/PWTP9-output.json

```diff
@@ -13730,6 +13732,10 @@
         "reference": "Specimen/0DB91FFF-DC9C-4246-A8D0-A3D340A2D822"
       } ],
       "result": [ {
+        "reference": "Observation/FA1B0FA9-C9BC-4399-B506-52D0286DFD06"
+      }, {
+        "reference": "Observation/E47265C2-CE59-4F24-B203-5A73CEB79D84"
+      }, {
```

Comment:
Multuiple batteries here, so multiple references. Which goes against what the GP Connect spec says.

Reply:
Have addressed this following our conversation with Martin.  There may be multiple batteries, but only one filing comment should be created to indicate the report as filed